### ISSUE TITLE
Segmentation fault when converting opus -> aac

### ISSUE THREAD

**Issue by kittenswolf** [enzo1982/freac#313]
**Describe the bug**
Note: I wasn't sure whether I should submit this issue here or on the BoCA repository, but since the issue occurs with the freaccmd CLI I posted it here.

Converting from opus to aac using the CLI tool results in a segmentation fault.

```
kittenswolf:~$ freaccmd -e fdkaac -o input.aac sample.opus
Segmentation fault (core dumped)
```

**To Reproduce**
1. Get any .opus file ([I uploaded the specific sample.opus file I'm testing with](https://kwl.sh/uploads/public/freac_issue_310.opus))
2. Execute `freaccmd -e fdkaac -o input.aac sample.opus`
3. See error

**Expected behavior**
I expected the file to be converted from opus to aac format.

**System (please complete the following information)**
 - OS: Ubuntu 22.04 LTS
 - CPU: AMD 3600
 - RAM: 64 GB

**Additional context**
My freac installation is manually built from commit 659094dced4e1f1d06c1017db674915b2b1915ba.

Package information:
```
kittenswolf:~$ apt show libopus-dev
Package: libopus-dev
Version: 1.3.1-0.1build2
Section: libdevel
Source: opus
Origin: Ubuntu
Depends: libopus0 (= 1.3.1-0.1build2)

kittenswolf:~$ apt show fdkaac
Package: fdkaac
Version: 1.0.0-1build1
Section: multiverse/sound
Origin: Ubuntu
Depends: libc6 (>= 2.14), libfdk-aac2 (>= 2.0.1)

```

**Comment by enzo1982** [comment]
Thank you for reporting this!

I was able to reproduce the issue after removing the MP4v2 library used by the FDK-AAC component to create MP4 AAC files (standard extension .m4a).

fre:ac creates MP4 AAC files by default and there was an issue causing the reported crash in freaccmd when the MP4v2 library is not available.

This is fixed now via enzo1982/boca@4541e5eff. With the fix, fre:ac creates ADTS AAC files (standard extension .aac) when the MP4v2 library is not available.

I'd recommend installing the MP4v2 library to enable fre:ac to create MP4 files. There do not seem to be Ubuntu packages for this library currently, so you will have to build and install it manually. It's available from [mp4v2.org](https://mp4v2.org).

> I wasn't sure whether I should submit this issue here or on the BoCA repository, but since the issue occurs with the freaccmd CLI I posted it here.

Yes, that's fine. Please post any issues that show up in fre:ac or freaccmd here.

**Comment by kittenswolf** [comment]
Thank you!
